intel/filterlists/updater.go

```diff
@@ -44,7 +46,11 @@ func performUpdate(ctx context.Context) error {
 	}
 	defer updateInProgress.UnSet()
 
-	module.Hint(filterlistsUpdateInProgress, filterlistsUpdateInProgressDescr)
+	module.Hint(
+		filterlistsUpdateInProgress,
+		"Filter Lists Update In Progress",
+		"The filter list system is processing updates. While this might slightly degrade performance, the filter list system stays functional during this process.",
+	)
```

Comment:
I would not talk about the "filter list system" as users don't know about internal systems/boundaries/modules. I'd just say: "The Portmaster is processing a filterlist update. While this might slightly degrade performance, the Portmaster is still protecting your privacy "